Multisig/UI/Safe Management/Create Safe/SafeCreatingViewController.swift

```diff
@@ -75,6 +75,7 @@ class SafeCreatingViewController: UIViewController {
         Timer.scheduledTimer(withTimeInterval: 3, repeats: false) { [weak self] timer in
             DispatchQueue.main.async {
                 self?.onSuccess()
+                self?.dismiss(animated: true)
```

Comment:
This fixes the bug, but there might be a better way

Reply:
I think this is misplaced: we put the dismiss of a view controller near the place where it was opened (on the calling side)

Reply:
same here, specially that we have onSuccess handler